### ISSUE TITLE
ExportToDatabase and ExportToSpreadsheet should use the same Image/Object Number column labels.

### ISSUE THREAD

**Issue by DavidStirling** [CellProfiler/CellProfiler#3995]
At present ExportToSpreadsheet outputs object ID under a column titled "ObjectNumber", but ExportToDatabase labels this column "Number_Object_Number". I think something similar happens with image numbers. Currently ExportToDb is grabbing the names from `M_NUMBER_OBJECT_NUMBER` in Measurement while ExportToSs uses it's own label. It might be better to make these headers consistent so that working on sheets exported from a database doesn't require scripts to be modified so much.

Also, is there a reason for prefacing Object_Number with "Number_"?

**Comment by braymp** [comment]
See my last answer to this post: https://forum.image.sc/t/trackobjects-data-columns/12958

**Comment by DavidStirling** [comment]
Right you are, it does look like Number_Object_Number is added to spreadsheets too. Are there scenarios where these might be different from ObjectNumber? If not we should probably try to eliminate the duplication.

**Comment by bethac07** [comment]
No, they wouldn't be different, but if we don't have the Number_Object_Number option people can't display the object number in the display modules (per Mark's linked comment above).  To get rid of the duplication at the spreadsheet level we either need to 

- Get rid of that ability (which I'm ok with, just pointing out is an option)
- Tell EtS/EtD to ignore that column (which I'm also ok with, depending how difficult/annoying it is)

**Comment by DavidStirling** [comment]
Sorry, I'm struggling to understand how Number_Object_Number relates to the display modules. In measurement.py we currently have definitions for "ObjectNumber", "Object_Number" and then "Number_Object_Number" (which itself is generated from combining "Number_" and "Object_Number"). Would it not be possible to make both export modules, and perhaps adapt references elsewhere, to use just one of these options?

**Comment by braymp** [comment]
IIRC, the Display modules (DisplayDataOnImage, etc) did not include ObjectNumber as a measurement to display in the UI dropdown. So we hacked in an extra measurement so that we would could display it. 

But I don't remember exactly why we couldn't/didn't just enable ObjectNumber as a displayable term or why it would have bad/wrong to do so. Whatever it was, it must have seemed like a good reason at the time. :-)
